Eastern-Research-Group/Trade-off_Tool_for_Sampling
# Feature/90 replace locate tab

## Related Issues:
* [TOTS-90](https://ergcloud.atlassian.net/browse/TOTS-90)

## Main Changes:
* Moved the locate feature to a widget on the map and removed the Locate tab.

## Steps To Test:
1. Verify the Locate tab is gone.
2. Verify a search widget is displayed in the top right of the map.
3. Verify the search works.
4. Switch to 3d view.
5. Repeat steps 2 and 3.
